react/LocationSearch.tsx

```diff
@@ -149,7 +164,8 @@ const LocationSearch: React.FC<LocationSearchProps> = ({
     }
 
     setSearchTerm(selectedAddress)
-    executeGetAddress({ variables: { id } })
+    executeGetAddress({ variables: { id, sessionToken } })
+    refetchSessionToken()
```

Comment:
Is there any drawback in using `useQuery` + `refetch` versus `useLazyQuery`? I'm just using it because I wanted the first query to be executed on startup.